packages/ui/src/components/Footer/index.tsx

```diff
@@ -49,78 +60,112 @@ const socialMediaIconTypes = {
-                                  external
-                                  href={item.href}
-                                  className={css('utrecht-link-list__link', {
-                                    'utrecht-link-list__link--phone': isPhoneNumber,
-                                  })}
-                                >
-                                  {!isPhoneNumber && <UtrechtIconChevronRight />}
-                                  <span className="utrecht-link-list__link-text">{item.textContent}</span>
-                                </Link>
-                              </GridCell>
-                            );
-                          })}
-                      </Grid>
-                    </li>
-                  ))}
-              </ul>
-            </GridCell>
-            <GridCell md={6} sm={6}>
-              {data?.address && <Markdown>{data?.address}</Markdown>}
-            </GridCell>
-          </Grid>
-        </GridCell>
-        <GridCell md={4} sm={2} className={css('utrecht-grid--justify-content-sm-flex-end')}>
-          <ul className={css('utrecht-link-list', 'utrecht-link-list--html-ul', 'utrecht-link-list--social-media')}>
-            {data?.socialMediaList &&
-              data.socialMediaList?.link.length > 0 &&
-              data.socialMediaList?.link.map((item, index) => {
-                const Icon = socialMediaIconTypes[item.icon as keyof typeof socialMediaIconTypes];
-                return (
-                  <li key={index} className={css('utrecht-link-list__item')}>
-                    <Link
-                      external
-                      href={item.href}
-                      className={css('utrecht-link-list__link')}
-                      aria-label={item.textContent}
-                    >
-                      <Icon />
-                      <span className="utrecht-link-list__link-text">{item.textContent}</span>
-                    </Link>
-                  </li>
-                );
-              })}
-          </ul>
-        </GridCell>
-      </Grid>
-    </Page>
-  </PageFooter>
+const SocialMediaNav = forwardRef(
+  (
+    {
+      children,
+      socialMediaLabel,
+      headingLevel = 3,
+      className,
+      headingRestProps,
+      ...restProps
+    }: PropsWithChildren<SocialMediaNavProps>,
+    ref: ForwardedRef<HTMLElement>,
+  ) => {
+    const headingId = socialMediaLabel ? useId() : undefined;
+
+    return (
+      <nav ref={ref} className={css(className, 'utrecht-social-media-nav')} aria-labelledby={headingId} {...restProps}>
+        {socialMediaLabel && (
+          <Heading
+            id={headingId}
+            className={css(headingRestProps?.className, 'utrecht-social-media-nav__heading')}
+            level={headingLevel}
+            aria-hidden="true"
+            {...headingRestProps}
+          >
+            {socialMediaLabel}
+          </Heading>
+        )}
+        {children}
+      </nav>
+    );
+  },
```

Comment:
Kan deze component niet generieker, en in een eigen file? Bijvoorbeeld gewoon `<Nav>`? Het hoeft toch niet social media of footer specifiek te zijn?